Cannot build cquery

```
Waf: Entering directory `/home/agauniyal/projects/cquery/build'
CLANG_INCLUDE_DIR: /home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/include
CLANG_LIB_DIR:     /home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/lib
[ 1/33] Compiling src/cache.cc
[ 2/33] Compiling src/clang_complete.cc
[ 3/33] Compiling src/clang_cursor.cc
[ 4/33] Compiling src/clang_index.cc
[ 5/33] Compiling src/clang_translation_unit.cc
[ 6/33] Compiling src/clang_utils.cc
[ 7/33] Compiling src/command_line.cc
[ 8/33] Compiling src/file_consumer.cc
[ 9/33] Compiling src/include_complete.cc
[10/33] Compiling src/indexer.cc
[11/33] Compiling src/ipc.cc
[12/33] Compiling src/ipc_manager.cc
[13/33] Compiling src/language_server_api.cc
../src/command_line.cc:2007:16: error: lambda capture 'working_files' is not used [-Werror,-Wunused-lambda-capture]
              [working_files, global_code_complete_cache,
               ^
../src/command_line.cc:3014:37: error: lambda capture 'config' is not used [-Werror,-Wunused-lambda-capture]
  WorkThread::StartThread("stdin", [config, request_times]() {
                                    ^
2 errors generated.

Waf: Leaving directory `/home/agauniyal/projects/cquery/build'
Build failed
 -> task in 'app' failed with exit status 1 (run with -v to display more information)
```

After fixing that -
```
Waf: Entering directory `/home/agauniyal/projects/cquery/build'
CLANG_INCLUDE_DIR: /home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/include
CLANG_LIB_DIR:     /home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/lib
[ 7/33] Compiling src/command_line.cc
[14/33] Compiling src/lex_utils.cc
[15/33] Compiling src/match.cc
[16/33] Compiling src/options.cc
[17/33] Compiling src/platform.cc
[18/33] Compiling src/platform_linux.cc
[19/33] Compiling src/platform_win.cc
[20/33] Compiling src/position.cc
[21/33] Compiling src/project.cc
[22/33] Compiling src/query.cc
[23/33] Compiling src/query_utils.cc
[24/33] Compiling src/serializer.cc
[25/33] Compiling src/standard_includes.cc
[26/33] Compiling src/task.cc
[27/33] Compiling src/test.cc
[28/33] Compiling src/third_party_impl.cc
[29/33] Compiling src/timer.cc
[30/33] Compiling src/utils.cc
[31/33] Compiling src/work_thread.cc
[32/33] Compiling src/working_files.cc
[33/33] Linking build/app
/usr/bin/ld: warning: libtinfo.so.5, needed by /home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/lib/libclang.so, not found (try using -rpath or -rpath-link)
/home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/lib/libclang.so: undefined reference to `setupterm'
/home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/lib/libclang.so: undefined reference to `tigetnum'
/home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/lib/libclang.so: undefined reference to `set_curterm'
/home/agauniyal/projects/cquery/build/clang+llvm-4.0.0-x86_64-linux-gnu-ubuntu-14.04/lib/libclang.so: undefined reference to `del_curterm'
clang-5.0: error: linker command failed with exit code 1 (use -v to see invocation)

Waf: Leaving directory `/home/agauniyal/projects/cquery/build'
Build failed
 -> task in 'app' failed with exit status 1 (run with -v to display more information)
```

--- a/src/command_line.cc
+++ b/src/command_line.cc
@@ -2004,7 +2004,7 @@ bool QueryDbMainLoop(Config* config,
           }
 
           ClangCompleteManager::OnComplete callback = std::bind(
-              [working_files, global_code_complete_cache,
+              [global_code_complete_cache,
                non_global_code_complete_cache, is_global_completion,
                existing_completion,
                msg](const NonElidedVector<lsCompletionItem>& results,
@@ -3011,7 +3011,7 @@ void RunQueryDbThread(const std::string& bin_name,
 // |ipc| is connected to a server.
 void LaunchStdinLoop(Config* config,
                      std::unordered_map<IpcId, Timer>* request_times) {
-  WorkThread::StartThread("stdin", [config, request_times]() {
+  WorkThread::StartThread("stdin", [request_times]() {
     IpcManager* ipc = IpcManager::instance();
 
     std::unique_ptr<BaseIpcMessage> message =

